packages/fuel-gauge/src/transaction-response.test.ts

```diff
@@ -200,59 +200,67 @@ describe('TransactionResponse', () => {
-      provider,
-    });
+      const destination = Wallet.generate({
+        provider,
+      });
 
-    const { id: transactionId } = await adminWallet.transfer(
-      destination.address,
-      100,
-      await provider.getBaseAssetId()
-    );
+      const { id: transactionId } = await adminWallet.transfer(
+        destination.address,
+        100,
+        await provider.getBaseAssetId()
+      );
 
-    const response = await TransactionResponse.create(transactionId, provider);
+      const response = await TransactionResponse.create(transactionId, provider);
 
-    const transactionSummary = await response.waitForResult();
-
-    expect(transactionSummary.id).toBeDefined();
-    expect(transactionSummary.fee).toBeDefined();
-    expect(transactionSummary.gasUsed).toBeDefined();
-    expect(transactionSummary.operations).toBeDefined();
-    expect(transactionSummary.type).toBeDefined();
-    expect(transactionSummary.blockId).toBeDefined();
-    expect(transactionSummary.time).toBeDefined();
-    expect(transactionSummary.status).toBeDefined();
-    expect(transactionSummary.receipts).toBeDefined();
-    expect(transactionSummary.mintedAssets).toBeDefined();
-    expect(transactionSummary.burnedAssets).toBeDefined();
-    expect(transactionSummary.isTypeMint).toBeDefined();
-    expect(transactionSummary.isTypeBlob).toBeDefined();
-    expect(transactionSummary.isTypeCreate).toBeDefined();
-    expect(transactionSummary.isTypeScript).toBeDefined();
-    expect(transactionSummary.isStatusFailure).toBeDefined();
-    expect(transactionSummary.isStatusSuccess).toBeDefined();
-    expect(transactionSummary.isStatusPending).toBeDefined();
-    expect(transactionSummary.transaction).toBeDefined();
-  });
+      const transactionSummary = await response.waitForResult();
+
+      expect(transactionSummary.id).toBeDefined();
+      expect(transactionSummary.fee).toBeDefined();
+      expect(transactionSummary.gasUsed).toBeDefined();
+      expect(transactionSummary.operations).toBeDefined();
+      expect(transactionSummary.type).toBeDefined();
+      expect(transactionSummary.blockId).toBeDefined();
+      expect(transactionSummary.time).toBeDefined();
+      expect(transactionSummary.status).toBeDefined();
+      expect(transactionSummary.receipts).toBeDefined();
+      expect(transactionSummary.mintedAssets).toBeDefined();
+      expect(transactionSummary.burnedAssets).toBeDefined();
+      expect(transactionSummary.isTypeMint).toBeDefined();
+      expect(transactionSummary.isTypeBlob).toBeDefined();
+      expect(transactionSummary.isTypeCreate).toBeDefined();
+      expect(transactionSummary.isTypeScript).toBeDefined();
+      expect(transactionSummary.isStatusFailure).toBeDefined();
+      expect(transactionSummary.isStatusSuccess).toBeDefined();
+      expect(transactionSummary.isStatusPending).toBeDefined();
+      expect(transactionSummary.transaction).toBeDefined();
+    }
+  );
+
+  it.todo(
```

Comment:
Was this forgotten?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3857/commits/2fc67ec0d3bcf3b32c4e3ecec83a75e4758ecdc4